### ISSUE TITLE
Add support for WordPress-generated databases

### ISSUE THREAD

**Issue by chelminski** [riflowth/skinsystem#115]
Can I count on you to do Wordpress integration in the future? A lot of websites on the Internet use the Wordpress engine, often server owners decide to synchronize player databases with wordpress (as in my case). I have such a problem that I do not use the classic AuthMe but Authme reloaded (allows players to log in with data from the wordpress website). My AuthMe reloaded configuration file is not working properly.

To sum up. Please add other authentication methods or integrate with Wodpress in the future. Sorry for my English
